apps/web/app/api/user/tokens/route.ts

```diff
@@ -31,34 +38,38 @@ export const GET = withSession(async ({ session }) => {
 
 // POST /api/user/tokens – create a new token for a specific user
 export const POST = withSession(async ({ req, session }) => {
-  const { name } = await req.json();
-  const token = nanoid(24);
-  const hashedKey = hashToken(token, {
-    noSecret: true,
-  });
-  // take first 3 and last 4 characters of the key
-  const partialKey = `${token.slice(0, 3)}...${token.slice(-4)}`;
-  await Promise.all([
-    prisma.token.create({
-      data: {
-        name,
-        hashedKey,
-        partialKey,
-        userId: session.user.id,
-      },
-    }),
-    qstash.publishJSON({
-      url: `${APP_DOMAIN_WITH_NGROK}/api/cron/notify`,
-      body: {
-        type: "API_KEY_CREATED",
-        props: {
-          email: session.user.email,
-          apiKeyName: name,
+  try {
```

Comment:
`withSession` acts as a global error handler for this API route. So `try catch` here may not be needed. Please remove this and see if you get the same response.

Reply:
Done.